dist/components/ui/msm-click-to-edit/msm-click-to-edit.scss

```diff
@@ -0,0 +1,37 @@
+.editable-text-container {
+    cursor: pointer;
+}
+
+.editable-text {
```

Comment:
But maybe there shouldn't be default styles for editable text as we could use this component for all kinds of texts (`<h1>, <small>, <span class=".full-name">`). What do you think?